### ISSUE TITLE
Lisfranc injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11155]
**Preferred term label:**
Lisfranc injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
Foot x-ray showing either widening of the space between the 1st and 2nd metatarsal to >2mm at the base, dorsal displacement of the base of the 1st or 2nd metatarsal, discontinuity of a line drawn from the medial base of the 2nd metatarsal to the medial side of the medial cuneiform, irregularity between the medial side of the 5th metatarsal base and the medial side of the cuboid, or disruption of the line connecting the medial aspect of the navicular bone to the medial cuneiform; lisfranc fracture-dislocation

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lisfranc injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Lisfranc injuries refer to bony or ligamentous compromise of the tarsometatarsal and intercuneiform joint complex and include a spectrum of injuries ranging from a stable, partial sprain to a grossly displaced and unstable fracture or fracture–dislocation of the midfoot.  Foot x-ray shows either widening of the space between the 1st and 2nd metatarsal to over 2mm at the base, dorsal displacement of the base of the 1st or 2nd metatarsal, discontinuity of a line drawn from the medial base of the 2nd metatarsal to the medial side of the medial cuneiform, irregularity between the medial side of the 5th metatarsal base and the medial side of the cuboid, or disruption of the line connecting the medial aspect of the navicular bone to the medial cuneiform.
PMID:[28188544](https://pubmed.ncbi.nlm.nih.gov/28188544/)


Comment: Ironically, the injury is named after Jaques Lisfranc de Saint-Martin, a French army field surgeon, who actually described a traumatic amputation through the midfoot. 

Syn: Lisfranc fracture-dislocation

**Comment by MickeySegal** [comment]
OK, but does it get an HPO code?

**Comment by pnrobinson** [comment]
@MickeySegal 

sorry, it is HP:0020352
